src/imitation/scripts/common/common.py

```diff
@@ -49,7 +49,7 @@ def hook(config, command_name, logger):
     if config["common"]["log_dir"] is None:
         env_sanitized = config["common"]["env_name"].replace("/", "_")
         log_dir = os.path.join(
-            "output",
+            config["common"]["log_root"] or "output",
```

Comment:
Should maybe define this as a separate variable since the logic is a little complex?

Reply:
yep